samples/account_summaries_list.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Analytics Admin API sample application which prints summaries of
+all accounts accessible by the caller.
+
+See https://developers.google.com/analytics/devguides/config/admin/v1/rest/v1alpha/accountSummaries/list
+for more information.
+"""
+# [START analyticsadmin_account_summaries_list]
+from google.analytics.admin import AnalyticsAdminServiceClient
+
+
+def run_sample():
```

Comment:
What's the purpose of this function?

Reply:
Removed occurrences of  run_samples() where the code is trivial.